src/opening-period/time-span/TimeSpan.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useEffect } from 'react';
+import { Button as HDSButton, IconTrash, Select, TextInput } from 'hds-react';
+import Weekdays from './Weekdays';
+import './TimeSpan.scss';
+import TimeInput from './TimeInput';
+import { ResourceStateOption } from '../../common/lib/types';
+
+export default function TimeSpan({
+  index,
+  remove,
+  register,
+  setValue,
+  resourceStateOptions,
+}: {
+  index: number;
+  remove: Function;
+  register: Function;
+  setValue: Function;
+  resourceStateOptions: ResourceStateOption[] | undefined;
```

Comment:
Should it be mandatory to have resourceStateOptions in this component? Otherwise we would not render it at all

Reply:
If I remove that `| undefined`, then typescript will complain in CreateNewOpeningPeriodPage.tsx this:

![image](https://user-images.githubusercontent.com/2777633/101782674-fc5ea800-3b01-11eb-8f8e-4318bfe6b1ec.png)